src/style.css

```diff
@@ -0,0 +1,20 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body div {
```

Comment:
create a common class for common style. 
Do not style tag, use class for this. Except `body`.